src/howard/howard/issuers/realisation.py

```diff
@@ -1,143 +1,159 @@
 """Realisation Certificate"""
 
+from datetime import date, datetime
 from enum import Enum
+from typing import Generic, TypeVar
+from uuid import UUID
 
 from django.template import Context
 
-from dateutil import parser
+import arrow
+from pydantic import BaseModel, ValidationError
 
 from marion.certificates.issuers.base import AbstractCertificate
 
+ArrowSupportedDateType = TypeVar("ArrowSupportedDateType", int, str, date)
 
-class CertificateScope(Enum):
-    """Allowed scopes for the RealisationCertificate"""
+
+class Gender(str, Enum):
+    """Gender definitions."""
+
+    MALE = "Mr"
+    FEMALE = "Mme"
+
+
+class CertificateScope(str, Enum):
+    """Allowed scopes for the RealisationCertificate."""
 
     FORMATION = "action de formation"
     BILAN = "bilan de compétences"
     VAE = "action de VAE"
     APPRENTISSAGE = "action de formation par apprentissage"
 
```

Comment:
Why not adding this to enums (also for gender), so that we are not forced to use `scope.value` but `scope` in the templates?
```suggestion
    def __str__(self):
        return f"{self.value}"
```